compiler/src/dmd/typesem.d

```diff
@@ -5295,7 +5295,7 @@ Type getComplexLibraryType(const ref Loc loc, Scope* sc, TY ty)
  * Returns:
  *     An enum value of either `Covariant.yes` or a reason it's not covariant.
  */
-Covariant covariant(Type src, Type t, StorageClass* pstc = null, bool cppCovariant = false)
```

Comment:
you will probably need to update `frontend.h` here too.

Reply:
Ah yes, the always annoying `frontend.h`. Done.